src/App.js

```diff
@@ -1,26 +1,33 @@
 import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import { BrowserRouter as Router, Route, NavLink } from 'react-router-dom';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['Router' is defined but never used.](https://app.codacy.com/gh/WebHeroSchool/test_react/pullRequest?prid=5584351)